TrkPatRec/src/RobustHelixFinderDiag_tool.cc

```diff
@@ -11,17 +11,25 @@
 
 #include "art/Utilities/ToolMacros.h"
 #include "art/Utilities/make_tool.h"
-#include "fhiclcpp/ParameterSet.h"
+#include "fhiclcpp/types/Atom.h"
+#include "fhiclcpp/types/Sequence.h"
 
 #include "TH1F.h"
 
 namespace mu2e {
 
   using namespace RobustHelixFinderTypes;
+  using RobustHelixFinderTypes::Config;
   
   class RobustHelixFinderDiag : public mu2e::ModuleHistToolBase {
   public:
 
+    // struct Config {
```

Comment:
It looks like this code is now in RobustHelixFinder_types, why is it entered here commented out?

Reply:
Hi Dave, I put this block here in RobustHelixFinderDiag_tool first, but Giani realized that it was meant to go in RobustHelixFinder_types to function properly. I just forgot to remove the commented block, it was a mistake that caused errors that we found on compilation.

Reply:
Fixed!